examples/info_files/_INFO_file_template_recommended.json

```diff
@@ -11,33 +11,33 @@
 	"checkpoints": [
     {
       "name": "Source",
-      "processStartTime": "01-01-1970 00:00:00 +0000",
-      "processEndTime": "01-01-1970 0:00:00 +0000",
+      "processStartTime": "<ProcessStartTime>",
```

Comment:
**For discussion:** I wonder if the same field name for the timestamps in `Source` and `Raw` doesn't imply they have to be the same? 🤔

Reply:
I will slightly change it